scripts/publish.sh

```diff
@@ -271,6 +277,23 @@ verify_ecr() {
 	verify_sha $sha1 $sha2
 }
 
+verify_ecr_image_scan() {
+	region=${1}
+	repo_uri=${2}
+	tag=${3}
+	
+	imageTag=$(aws ecr list-images  --repository-name ${repo_uri} --region ${region} | jq -r '.imageIds[].imageTag' | grep -c ${tag} || echo "0")
+	if [ "$imageTag" = '1' ]; then
+		aws ecr start-image-scan --repository-name ${repo_uri} --image-id imageTag=${tag} --region ${region}	
+		aws ecr wait image-scan-complete --repository-name ${repo_uri} --region ${region} --image-id imageTag=${tag}
+		vulnerabilityCount=$(aws ecr describe-image-scan-findings --repository-name ${repo_uri} --region ${region} --image-id imageTag=${tag} | jq '.imageScanFindings.findings | length')
+		if [ "$vulnerabilityCount" != '0' ]; then
+			echo "Uploaded image ${tag} has ${vulnerabilityCount} vulnerabilities."
+			exit 1
```

Comment:
Looking at the docs: https://docs.aws.amazon.com/cli/latest/reference/ecr/describe-image-scan-findings.html

You get this:
```
"findingSeverityCounts": {
          "MEDIUM": 1
      }
```

So we can do per-severity counts, and only block certain severities. (Not sure what we should restrict it to- may be high and critical?)

Reply:
Agreed, as per this https://nvd.nist.gov/vuln-metrics/cvss, I think blocking high and critical should be good.